database/migrations/2019_11_29_202342_create_playlist_songs_table.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreatePlaylistSongsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('playlist_songs', function (Blueprint $table) {
```

Comment:
https://laravel.com/docs/6.x/eloquent-relationships#many-to-many

We should follow laravel naming conventions and name that table `playlist_song`. Remember to change class and file name too